content/documentation/guides/events.es.md

```diff
@@ -0,0 +1,250 @@
+---
+name: Eventos
+seoDescription: Escuchando por eventos en una aplicación de Buffalo
+seoKeywords: ["buffalo", "go", "golang", "events", "plugins"]
+aliases:
+  - /docs/events
+  - /es/docs/events
+---
+
+# Eventos
+
+{{< since "0.13.0-beta.2" >}}
+
+La librería de {{< doclink href="github.com/gobuffalo/events" message="events" >}} permite a las aplicaciones en Go, incluyendo aplicaciones de Buffalo, escuchen y emitan mensajes de eventos globales.
+
+## Escuchar Eventos
+
+Para iniciar a escuchar eventos, primero se debe registrar un {{< doclink message="events#Listener" href="github.com/gobuffalo/events#Listener" >}} con la librería {{< doclink href="github.com/gobuffalo/events" message="events"  >}}.
+
+```go
+func init() {
+  _, err := events.Listen(func(e events.Event) {
+    // do work
+  })
+}
+```
+
+Una vez registrada esta nueva función `listener`, se enviarán todos los eventos emitidos a través de la librería {{< doclink message="events" href="github.com/gobuffalo/events" >}}.
+
+## Emitiendo Eventos
+
+Cuando emitimos eventos, el atributo `Kind` debe ser una cadena única, pero constante. Es este atributo que los usuarios usarán para determinar cómo responder a los eventos que reciben.
+
+Se recomienda asignar un espacio de nombres a este atributo como tal, con los eventos de error sean con el sufijo: `:err`.
+
+```plain
+"<package-name>:<additional-names>:<optional-error>"
+
+"myapp:foo:start"
+"myapp:foo:stop"
+"mypkg:workers:bar:err"
+```
+
+Este patrón de nomenclatura facilita a los usuarios filtrar solo aquellos eventos que les interesan. Ve a [Filtrando Eventos](#filtering-events) para mas detalles.
+
+---
+
+Hay varias maneras de emitir un {{< doclink message="events#Event" href="github.com/gobuffalo/events#Event" >}} en tu código de Go. LAs funciones {{< doclink message="events#EmitError" href="github.com/gobuffalo/events#EmitError" >}} y {{< doclink message="events#EmitPayload" href="github.com/gobuffalo/events#EmitPayload" >}} aceptan un argumento de tipo `payload interface{}`. Se recomienda usar {{< doclink message="events#Payload" href="github.com/gobuffalo/events#Payload" >}} para datos de entrada; cualquier otro tipo pasado será convertido a {{< doclink message="events#Payload" href="github.com/gobuffalo/events#Payload" >}} con el argumento establecido en los datos de entrada con la llave `data`.
```

Comment:
In "LAs", two first letters are capitalized

Reply:
Good catch @YanDeLeon , fixing